spec/support/example_app_helper.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+
+module ExampleAppHelper
+  def create_example_app
+    FileUtils.cd(tmp_path) do
+      `rails new #{app_name} -d postgresql --skip-action-mailer --skip-action-mailbox --skip-action-cable --skip-sprockets --skip-listen --skip-javascript --skip-turbolinks --skip-system-test --skip-test-unit --skip-bootsnap --skip-spring`
+    end
+  end
+
+  def remove_example_app
+    FileUtils.rm_rf(example_app_path)
+  end
+
+  def install_good_job
+    File.open("#{example_app_path}/Gemfile", 'a') do |f|
+      f.puts "gem 'good_job'"
+    end
+
+    FileUtils.cd(example_app_path) do
+      `rails g good_job:install`
+    end
```

Comment:
I think this could be turned into a generic function like:
```suggestion
    def example_app_cmd(*args)
      FileUtils.cd(example_app_path) do
        system(*args) || abort("\n== Command #{args} failed ==")
      end
    end
```